lib/hipmer/hipmerUtils.py

```diff
@@ -70,6 +70,25 @@ def _validate_inputs(self, params):
         # just in case I need the list someday.
         params['mer_sizes_int'] = mer_sizes_int
 
+        # Parse the scaff kmer string into a list.  This step verifies that there
+        # were integers separated by commas. I don't actually use the created list (scaff_mer_sizes_int).
+        if params.get('scaff_mer_lens'):
```

Comment:
Should we have a default if this were empty?

Reply:
@scanon there's a default in spec.json if someone invokes from narrative.  you're right though, we should message if it's not set.  I want call to include explicit params, so not setting val with default in code, but am recommended default vals.  OK?